Documentation/running-exporters.md

```diff
@@ -0,0 +1,66 @@
+# Running Exporters
+Running exporters and scraping them with prometheus-operator.
+
+## Two approaches to utilizing ServiceMonitors
```

Comment:
Which is why I commented on the lower point of how to use `ServiceMonitor`s. It's better to tell users the recommended way and people are more likely to use it that way and once they fully understand what's happening and consciously decide for the other way it's fine.